QtWebApp/httpserver/httplistener.h

```diff
@@ -91,10 +91,9 @@ class QTWEBAPP_EXPORT HttpListener : public QTcpServer {
 	/**
 	  Sent to the connection handler to process a new incoming connection.
 	  @param socketDescriptor references the accepted connection.
-	*/
-	
-	void handleConnection(tSocketDescriptor socketDescriptor);
-	
+        */
+        void handleConnection(qintptr socketDescriptor);
+
```

Comment:
Can you fix the indentation please? This is supposed to be indented with one tab.